tutorials/2d/2d_sprite_animation.rst

```diff
@@ -198,6 +220,27 @@ released.
         else:
             $AnimationPlayer.stop()
 
+ .. code-tab:: csharp
```

Comment:
Shouldn't this block be about `AnimationPlayer` instead of repeating the `AnimatedSprite` one?